extensions/email/src/test/java/org/mule/extension/email/POP3RetrieverTestCase.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.extension.email;
+
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsInstanceOf.instanceOf;
+import static org.junit.Assert.assertThat;
+import org.mule.runtime.api.message.MuleMessage;
+import org.mule.runtime.core.api.MuleEvent;
+
+import java.util.List;
+
+import org.junit.Test;
+
+public class POP3RetrieverTestCase extends AbstractEmailRetrieverTestCase
```

Comment:
why is it that the IMAP version has more tests? DIdn't you said that the only difference between pop and imap is the operations locality and the content fetching? Why doesn't the rest apply?